src/Core/src/Handlers/Window/WindowHandler.Windows.cs

```diff
@@ -4,7 +4,7 @@ namespace Microsoft.Maui.Handlers
 {
 	public partial class WindowHandler : ElementHandler<IWindow, UI.Xaml.Window>
 	{
-		RootPanel? _rootPanel = null;
+		internal RootPanel? _rootPanel = null;
```

Comment:
Why internal now? Maybe we need a more predictable way? For example, maybe in the init, get the window content and use that. That way, we don't directly depend on the handler internal implementation. Especially if someone can swap out the handler for a custom one.